htdocs/modules/system/class/block.php

```diff
@@ -325,11 +325,11 @@ public function insert(XoopsObject $obj, $force = true)
     /**
      * retrieve array of {@link XoopsBlock}s meeting certain conditions
      *
-     * @param  CriteriaElement $criteria  {@link CriteriaElement} with conditions for the blocks
-     * @param  bool            $id_as_key should the blocks' bid be the key for the returned array?
-     * @param  bool            $as_object return an array of objects
+     * @param CriteriaElement $criteria {@link CriteriaElement} with conditions for the blocks
+     * @param bool $id_as_key           should the blocks' bid be the key for the returned array?
+     * @param bool $as_object           return an array of objects
      *
```

Comment:
This set of changes does not add value. It changes comments that did follow a real standard (PEAR) to follow nothing.